### DISCUSSION TITLE
VFO frequency scan limits.

### DISCUSSION THREAD

**Discussion by oisact** [AubsUK/uv-k5-firmware-custom#11] [Q&A]
**Disregard** - I found that long-pressing 5 / NOAA scans the range between VFO-A to VFO-B.

I've searched the original radio documentation, ezgumer, etc, and can't find any information on frequency scan ranges. That is, the ability to set a lower and upper limit, and frequency scanning will stay within that range.  Is this a capability of the radio? If not it may be a good feature to add to this fork, as this particular version already focuses heavily on scanning features.

**Comment by AubsUK** [comment]
I've never used it, but Egzumer's manual has instructions on setting ranges when scanning VFO:
https://github.com/egzumer/uv-k5-firmware-custom/wiki/Radio-operation#frequency-scanning

Edit - Just saw you found it :)
